### PR TITLE
Upgrade xorm to v0.8.0

### PR DESCRIPTION
**Pull request by lunny**
~blocked by https://github.com/lafriks/xormstore/pull/12~

### CODE REVIEW

**Comment by lafriks** [comment]
@lunny merged and tagged lafriks/xormstore `v1.3.2`

**Comment by lunny** [comment]
@lafriks done.

**Comment by lafriks** [comment]
gofmt failure on ci

**Comment by lunny** [comment]
@lafriks done.

File: go.sum [hunk 1]
```diff
@@ -811,7 +801,7 @@ strk.kbt.io/projects/go/libravatar v0.0.0-20191008002943-06d1c002b251 h1:mUcz5b3
 strk.kbt.io/projects/go/libravatar v0.0.0-20191008002943-06d1c002b251/go.mod h1:FJGmPh3vz9jSos1L/F91iAgnC/aejc0wIIrF2ZwJxdY=
 xorm.io/builder v0.3.6 h1:ha28mQ2M+TFx96Hxo+iq6tQgnkC9IZkM6D8w9sKHHF8=
 xorm.io/builder v0.3.6/go.mod h1:LEFAPISnRzG+zxaxj2vPicRwz67BdhFreKg8yv8/TgU=
-xorm.io/core v0.7.2-0.20190928055935-90aeac8d08eb h1:msX3zG3BPl8Ti+LDzP33/9K7BzO/WqFXk610K1kYKfo=
-xorm.io/core v0.7.2-0.20190928055935-90aeac8d08eb/go.mod h1:jJfd0UAEzZ4t87nbQYtVjmqpIODugN6PD2D9E+dJvdM=
 xorm.io/core v0.7.2 h1:mEO22A2Z7a3fPaZMk6gKL/jMD80iiyNwRrX5HOv3XLw=
```
Comment (guillep2k) [inline go.sum:804]: Is it OK for 0.7.2 to be still here?

Comment (lunny) [reply go.sum:804]: @guillep2k That's OK. It's `xorm.io/core` not `xorm.io/xorm`.

**Review by guillep2k — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

**Review by sapk — APPROVED** [review]

**Pull request merged**